subsys/bluetooth/audio/csip_set_member.c

```diff
@@ -492,10 +492,11 @@ static void csip_security_changed(struct bt_conn *conn, bt_security_t level,
 static void handle_csip_disconnect(struct bt_csip_set_member_svc_inst *svc_inst,
 				   struct bt_conn *conn)
 {
-	LOG_DBG("Non-bonded device");
-	if (is_last_client_to_write(svc_inst, conn)) {
+	if (is_last_client_to_write(svc_inst, conn) &&
```

Comment:
Hmm, I think I remember this function only being called for non-bonded devices (hence the "non-bonded device" log statement). 

Since the function is a no-op for bonded devices, we could either add the `bt_addr_le_is_bonded` check at the very beginning and return early, or we could guard the call to `handle_csip_disconnect` by `bt_addr_le_is_bonded`

I think that will be a clear solution that checking `!bt_addr_le_is_bonded` twice

Reply:
Done.